src/App/component/AddProjectDialog.jsx

```diff
@@ -15,32 +13,22 @@ import {
 
 export default function AddProjectDialog({open, reloadProjects, handleClose}) {
   const [projectName, setProjectName] = useState("")
-  const [githubRepositoryURL, setGithubRepositoryURL] = useState("")
-  const [sonarRepositoryURL, setSonarRepositoryURL] = useState("")
-  const setIsGithubAvailable = useState(false)[1]
-  const setIsSonarAvailable = useState(false)[1]
   const jwtToken = localStorage.getItem("jwtToken")
 
   const createProject = () => {
     let checker = []
-    if (projectName === "" || (githubRepositoryURL === "" && sonarRepositoryURL === "")) {
+    if (projectName === "") {
       alert("不準啦馬的>///<")
     } else {
-      if (githubRepositoryURL !== "") {
-        checker.push(checkGithubRepositoryURL());
-      }
-      if (sonarRepositoryURL !== "") {
-        checker.push(checkSonarRepositoryURL());
-      }
 
       Promise.all(checker)
         .then((response) => {
           if (response.includes(false) === false) {
             let payload = {
               projectName: projectName,
```

Comment:
Suggest also help remove the unneeded map-value since ES6 have the [Object property shorthand feature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer).
```suggestion
              projectName,
```